bedrock/settings/base.py

```diff
@@ -2041,7 +2041,7 @@ def JINJA_CONFIG():
     'bedrock.mozorg.context_processors.facebook_locale',
     'bedrock.firefox.context_processors.latest_firefox_versions',
     'jingo_minify.helpers.build_ids',
-))
+), exclude=('session_csrf.context_processor',))
```

Comment:
I did. I didn't add "session_csrf" as a dependency since we don't use it, but unless I exclude it here funfactory will helpfully add it to context processors which will try and fail to import it.